examples/all-bbes.json

```diff
@@ -222,6 +222,10 @@
             {
                 "name": "Let Expression",
                 "url": "let-expression"
+            },
+            {
+                "name": "Never Type",
+                "url": "never-type"
```

Comment:
Shall we add this after the `any` type, so that the types are grouped together?

Reply:
+1

Reply:
okay